public/css/modals/application-review-modal.css

```diff
@@ -1,5 +1,24 @@
+/* FULL MODAL */
 .modal-content {
-	background-color: #fafafa;
+	background-color: var(--white);
+}
```

Comment:
You can move this change to `modal.css` because we'll want it across the whole website.

While you're at it, a couple more things about the modal. Remove the padding from `.modal-content` and set the `border-radius` to 0 in `modal.css`. We want sharp corners and no white border on all of our modals.

Reply:
I'm getting rid of the `border: 1px solid #888` as well as otherwise it clashes with the dark theme section of the modal.

![image](https://user-images.githubusercontent.com/57082175/131947767-cb60e738-4b07-43e2-8f55-f347a87b2a4f.png)